src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/CCJStaffNotificationService.java

```diff
@@ -60,7 +60,7 @@ private EmailData prepareEmailData(final Claim claim) {
             staffEmailProperties.getRecipient(),
             emailContent.getSubject(),
             emailContent.getBody(),
-            singletonList(generateCcjPdf(claim))
+            singletonList(generateCountyCourtJudgmentPdf(claim))
```

Comment:
The PR title doesn't seem to reflect what is actually going on

Reply:
??

Reply:
we want to take details from CCJ and not from claim as done previously, that is exactly what we are doing in this PR

Reply:
'Take defendant details from ccj rather than claim'
The change made doesn't change any behaviour at all.

The real change in this PR is to change how `amountToPayByDefendant` is calculated

Reply:
I've renamed it